api-server/Web/Controllers/GitHubController.cs

```diff
@@ -1547,7 +1550,7 @@ public async Task<ActionResult> FilterUserPRs([FromBody] PRFilter filter)
     }
 
     [HttpPost("prs/waitingauthor/filter")]
-    public async Task<ActionResult> FilterWaitingAuthors([FromBody] PRFilter filter)
+    public async Task<ActionResult> FilterWaitingAuthors([FromBody] PRFilter filter, int page = 1)
```

Comment:
<!--Using HubReview-->
thought(if-minor): This filter usually returns fewer objects than a page size. There might be no need for paging in this filter.

Reply:
You're right, this system is redundant here. I will remove it